### ISSUE TITLE
MonoDevelop Won't Run

### ISSUE THREAD

**Issue by rhinox202** [lextm/monodevelop-windows#3]
I recently stumbled across your MonoDevelop repository on GitHub. After installing the 2 requirements and MD, I am unable to launch it. The process opens briefly and then exits. Any thoughts? Been looking for a Windows build for a while. Let me know if I can provide more information. Thanks.

**Comment by topeterk** [comment]
It runs on my end but I was able to build latest version. Make sure you have installed all the prerequisites, like gettext, GTK or MonoLibraries.

**Comment by rhinox202** [comment]
I got around to testing this again. Not sure what happened but it is working now. Might be because I ran it under Windows Sandbox initially. Anyway, I installed GTK-Sharp 2.12.45, Visual Studio Build Tools 2017 (link from Chocolately), and .NET Framework 4.7 Developer Pack (NDP47-DevPack-KB3186612-ENU). Everything is working great. I'm coding with WinForms, so there isn't any GUI editor, but debug and IntelliSense work perfectly.

Thank you to whoever works on this project! I'm coming from SharpDevelop which is awesome but has some quirks.
